sections/exchange/hooks/useExchange.tsx

```diff
@@ -978,6 +990,8 @@ const useExchange = ({
 		<>
 			{!isWalletConnected ? (
 				<ConnectWalletCard attached={footerCardAttached} />
+			) : hasNoL2Gas ? (
+				<GetL2GasCard attached={footerCardAttached} />
```

Comment:
keep wondering why we need a L2 gas card

Reply:
or at least why does it have to be here

Reply:
So the reason we have a GetL2GasCard is because this is a whole separate component. In the event the wallet is connected and we see your wETH balance is empty we display a component that directs users to the Optimism bridge.

Reply:
should this type of component also be relevant to mainnet as well (with ETH instead of WETH)? I know most people who go far enough into defi and find Synthetix have ETH, but we shouln't restrict this check to L2